composer.json

```diff
@@ -5,7 +5,7 @@
     "license": "MIT",
     "require": {
         "php": ">=8.1",
-        "cycle/database": "^2.5",
+        "cycle/database": "^2.6.2",
```

Comment:
I think it will be 2.7 because there a new behavior will be added.
I mean the `blob to varbinary transformation` feature